paper/paper.md

```diff
@@ -34,17 +34,17 @@ In this package, the ACCA algorithm has been extended to work directly with corr
 
 The `corrp` package is an R package that provides a flexible and efficient way of performing correlation-like analysis on mixed-type data frames. These datasets can contain different variable types, such as continuous (numeric), ordinal (ordered categorical), and nominal (unordered categorical) variables, which frequently arise in practical scenarios.
 
-Moreover, most traditional correlation methods in R are applicable only to specific data types or small datasets. In this sense, `corrp` extends this capability by handling mixed data types, integrating various association methods, and offering clustering directly from the resulting correlation matrix.
+Moreover, most traditional correlation methods in R apply only to specific data types or small datasets. In this sense, `corrp` extends this capability by handling mixed data types, integrating various association methods, and offering clustering directly from the resulting correlation matrix.
```

Comment:
You should mention here some of the "most traditional correlation methods" and provide references.